manager/src/main/java/com/redhat/service/smartevents/manager/BridgesService.java

```diff
@@ -30,4 +31,8 @@ public interface BridgesService {
     BridgeDTO toDTO(Bridge bridge);
 
     BridgeResponse toResponse(Bridge bridge);
+
+    Long getActiveBridgeCount(String orgId, QuotaType instanceType);
+
+    boolean isBridgeActive(String id);
```

Comment:
what is an `active` bridge?

Reply:
I will add the method docs. As a abstract active bridges are those which are not expired. Bridge expiration time is calculated using duration value define in `service-limit.json` file

Reply:
done

Reply:
make sense

Reply:
done